src/Umbraco.Core/PropertyEditors/BlockListEditor.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Umbraco.Core.PropertyEditors
+{
+    public class BlockListEditor
```

Comment:
@Shazwazza intended to be empty?

Reply:
Doh, will delete, not supposed to be there